src/client.rs

```diff
@@ -485,6 +485,11 @@ impl<C: Config> Client<C> {
         }
     }
 
+    /// Sets the action bar for this client.
+    pub fn set_action_bar(&mut self, text: impl Into<Text>) {
+        self.send_packet(SendActionBar { text: text.into() });
```

Comment:
Might be better to defer sending this packet.

The reason is that the login/join game packet is also deferred, and a lot of packets don't work correctly if they are sent before the login packet.

The other reason is that if set_action_bar is called multiple times in a single tick, you'll only send one packet. But then you have to add a field to `Client`.

But whatever this works fine. Just something to think about.